app/serializers/poster_serializer.rb

```diff
@@ -2,4 +2,14 @@
 
 class PosterSerializer < BasicUserSerializer
   include UserPrimaryGroupMixin
+
+  attributes :external_id
```

Comment:
Since `PosterSerializer` is inheriting from `BasicUserSerializer`, do we need to re-add the `external_id`?